includes/actions.php

```diff
@@ -245,3 +245,23 @@ function give_set_donation_levels_max_min_amount( $form_id ) {
 }
 
 add_action( 'give_pre_process_give_forms_meta', 'give_set_donation_levels_max_min_amount', 30 );
+
+/**
+ * This function will clear seo sitemap cache on update of settings
+ *
+ * @since 1.8.9
+ *
+ * @return void
+ */
+function give_clear_seo_sitemap_cache_on_settings_change() {
```

Comment:
@mehul0810 I have following suggestions:
1. Clear sitemap cache only when the only form related settings change.
    for example, single form pageview setting edit or form archive setting edit
2. Add `is_admin` check and move this code to `admin/admin-actions.php`